docker/Dockerfile.kinesis

```diff
@@ -0,0 +1,48 @@
+ARG DISTRO=focal
+ARG CORE_BUILDER_IMAGE=abxit/kinesis-core-builder:latest
+
+##########################################################################################################
+# # Build stage, includes everything needed to build.
+FROM ${CORE_BUILDER_IMAGE} as buildstage
+ENV DEBIAN_FRONTEND=noninteractive
+
+ARG NPROC=2
+ARG BUILD="enable-tests"
+
+COPY . /stellar-core
+
+WORKDIR /stellar-core
+RUN git submodule init && git submodule update
+RUN  ./docker/ci-kinesis "${BUILD}" ${NPROC}
+
+##########################################################################################################
+# replace stellar-core
+FROM ubuntu:${DISTRO}
+
+EXPOSE 11625
+EXPOSE 11626
+
+VOLUME /data
+VOLUME /postgresql-unix-sockets
+
+ADD docker/setup /setup.sh
+RUN ./setup.sh
+
+# We need to repeat ARG here to make it available inside build context
+# See https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
+ARG DISTRO=focal
+ARG STELLAR_CORE_VERSION=18.5.0-877.d387c6a71.focal
```

Comment:
version bump

Reply:
fixed in https://github.com/bullioncapital/kinesis-core/pull/24/commits/c9b8e4202d724246c742c2696e14a756b35616c7